src/program.fs

```diff
@@ -12,7 +12,7 @@ namespace Elmish
 type Program<'arg, 'model, 'msg, 'view> = private {
     init : 'arg -> 'model * Cmd<'msg>
     update : 'msg -> 'model -> 'model * Cmd<'msg>
-    subscribe : 'model -> Cmd<'msg>
+    subscribe : 'model -> Sub<'msg> list
```

Comment:
I love symmetry. However, I always felt cognitive dissonance and confusion with Cmd in Elm. It's name is singular but it represents multiple items. But then sometimes it behaves singular, like `Task.perform` resulting in a Cmd. One of the first reasons I looked at Elm source code.

In Elmish, the Cmd type alias always felt in the way, unnecessarily hiding the list. If it wasn't such a big change, the best thing IMO would be to drop the list alias and rename Effect to Cmd.

Reply:
If we make Sub like Cmd, we need to come up with another name for a single Sub item. I suggest `Svc<'msg>` since it's essentially a service with start and stop behavior. In JS, that's mainly about listening to DOM events. But in dotnet, it could be a long-running Task or Async on the thread pool.

Reply:
For public signature, there are two audiences for the Sub type. One is library authors. Another is end users who need to create their own subscriptions. (The most likely scenario until libraries emerge.)

Anonymous record seems problematic here. A library/user fn that needs to adjust a Sub item in some way would have to repeat the record definition in a type annotation. And it lacks some record features, namely destructuring. If we're going for parity, this would also be different from what we do with Cmd, where an individual item has a type definition, even if it's an alias. (To eliminate type defs, a more readable choice might be to define a record wrapper for the fn.)